feat: Introducting Elestio as fully managed deployment option

kaiwalyakoparkar opened the pull request:
closes #314

kevinpapst commented:
Be prepared to get roasted if you think you can get a backlink that easy: no. That is not enough.

All I see is that your SEO / Marketing team did setup a template page for OSS software that is not even made with love but with copy&paste from my website.

You need to add way more infos (be creative with your text!):

Where are your data centers?
How to pay? 
What makes Elestio special?
Why would anyone use Elestio?
Where is your team located?
How can users of Kimai install plugins? Via support or do they have full SSH/bash access?

The URLs of all links are wrong:
<img width="1405" alt="Bildschirmfoto 2023-10-20 um 17 20 27" src="https://github.com/kimai/www.kimai.org/assets/533162/72141726-dc39-43b5-aeba-ed5b3ee3d779">

This will only be added if you can make it beneficial for my users.

kaiwalyakoparkar commented:
Hey @kevinpapst, Thank you so much for the suggestions. I can assure you that Elestio provides value to Kimai users. I have sent you an email regarding the information you mentioned above. Looking forward to the collaboration :)

kaiwalyakoparkar commented:
@kevinpapst, I will be adding the answers to you abouve questions here to add additional context to this conversation
- Elestio offer fully managed instances of Kimai in 112 regions / 28 countries. We take care of deployment, security, dns, smtp, SSL, monitoring/alerts, backups, automated updates, migrations and a team of Linux experts to help you.
- Elestio is used by more than 1000 customers around the world from early startups to larger enterprises
- Our HQ and biggest part of the team is in Europe (Ireland & France mostly), rest of the team is in India, Israel & Japan.
- Users get full root SSH access to the service for advanced customizations, it's also possible to change the stack & env vars from our web dashboard.

kevinpapst commented:
Does you system support Kimai updates or ONLY the installation and the updates are then executed manually?

Please rebase your PR and move the info to one of the two new pages: shared hosting or the one for 1-click installations.

kaiwalyakoparkar commented:
@kevinpapst Yes indeed we do automated updates for Kimai, by default it's done every Sunday at 4am, but this can be configured or disabled from our dashboard. I will be working on the rebase of the pull request soon 🙌

kevinpapst reviewed (changes requested):
1. Remove package-lock.json 

2. You are a big company and you already shared info. Add it to the docs page. 
What do users get when using your system? 
Why should they use you?
What do you offer?
Who are you?

The current content is not helpful. When a user has the intend to get a new hoster, why would they choose your from that list?This is pure link dropping and I will NOT merge it until you fix that. If you don't know how to rewrite it, ask your marketing team.

kaiwalyakoparkar commented:
So sorry for that @kevinpapst I assumed you wanted changes on the Elestio website (Our team is working on content for the website). I have added the information here and removed the package-lock file. Happy to do updates as required :)

kevinpapst commented:
Why did you delete the package lock file? Did you really think I meant to remove it from the project?
Remove this change. If you donÄt know how, create a new PR.

The content is better. Now add some infos what do users get in regard to Kimai:
 Installation, updates => both are only one-click?

kaiwalyakoparkar commented:
@kevinpapst I apologise for the misunderstanding! I have made the suggested changes.

Installation info => Few clicks for the users to select cloud service providers and other configurations of their choice along with the Kimai version of their choice
Update info => Updates are carried out weekly automatically, but that can be configured or disabled in the dashboard

kevinpapst reviewed (approved):
Thank you @kaiwalyakoparkar - thank took a while 😁 but I like the result 👍
